tests/FileHelperTest.php

```diff
@@ -38,14 +48,21 @@ private function isChmodReliable(): bool
         return $mode === '0700';
     }
 
-    public function tearDown()
+    /**
+     * TearDown.
+     *
+     * @return void
```

Comment:
Such phpdoc is not useful so could be removed.

Reply:
Done.